### DISCUSSION TITLE
What function do I use to export collection names from Zotero to Obsidian?

### DISCUSSION THREAD

**Discussion by sicahjes** [retorquere/zotero-better-bibtex#2808] [Q&A]
_Firstly, I'm brand spanking new to bibtex, so I know this is likely a rudimentary question but I've been unable to figure out the solution. I've searched here (and online, read the guide etc.) but I think I'm held back by my limited understanding._

I have set up a workflow in Obsidian that uses a template to export data from Zotero to Obsidian in a literature note. At the top of this page, I have the basic bibliographic data, using functions to pull it from Zotero. However, I cannot get it to display collection names. I have tried {{collections}}, {{itemcollections}}, {{item.collections}}, {{collections | name}},  {{item.collections | name}}, {{item.collectionsn}}, {{item.collections0name}} etc. 

When I used the data explorer, I can see that the collections section has additional fields (a number, full path, key, and name) but I can't figure out how to use these terms to get it to work. Everything I've tried either results in a blank output for that function, or an export error.

Basically, I'd just love to get it to work the same as tags, so that it lists all of the collection names the item is associated with. For example, a source that was in two collections (one called literature review and one called key theory) would result in the below output:

**Collections**: literature review, key theory 

ps: I started by using the [Better Than Bibtex for Zotero Guide](https://retorque.re/zotero-better-bibtex/citing/index.html), but I am so new to this it would be handy to have a further simplified cheat sheet. Recommendations welcome.

**Comment by retorquere** [comment]
I can't help you with this -- I don't use Obsidian and I don't know what it calls in BBT.
